config/components.yml

```diff
@@ -54,10 +54,11 @@ frameworks:
   - "JavaBuildpack::Framework::Debug"
   - "JavaBuildpack::Framework::DynatraceOneAgent"
   - "JavaBuildpack::Framework::ElasticApmAgent"
-#  - "JavaBuildpack::Framework::GoogleStackdriverDebugger"
+# - "JavaBuildpack::Framework::GoogleStackdriverDebugger"
```

Comment:
you could even remove this line ?

Reply:
Yes we will after some time, this just makes it easier if someone wants to fork the buildpack and re-enable the framework.